src/components/hero/__snapshots__/hero-summary.test.js.snap

```diff
@@ -26,7 +30,15 @@ exports[`HeroSummary renders correctly 1`] = `
     <div
       className="summaryText"
     >
-      We are ESMA and we’re tackling social mobility by providing you with a centralised platform to house statistics, events, case studies and much more.
+      <span
+        className="highlight"
+      >
+        Social inequality
+      </span>
+       is persistent and systematic. Often where you start in life is where you’ll end up.
+      <br />
```

Comment:
these paragraphs should be separate `<p>` – avoid using `<br />` for styling like this